examples/undocumented/python/graphical/converter_ffsep_bss.py

```diff
@@ -7,12 +7,9 @@
 Kevin Hughes 2013
```

Comment:
question: could we group all the ica examples into one? The code is more or less the same. We could just have a loop over all the different ica methods